config_helper.py

```diff
@@ -0,0 +1,300 @@
+                           "The only valid options are: str, int, float, bool")
+
+def getcolor(option, card_type=None, card_name=None):
+    option = "color_"+option if option else "color"
+    return _get_list_from_config(option, card_type, card_name)
+
+def getanchor(option, card_type=None, card_name=None):
+    option = "anchor_"+option if option else "anchor"
+    anchor = _get_list_from_config(option, card_type, card_name)
+    if not (isinstance(anchor, (list, tuple)) and len(anchor) == 2):
+        raise AnchorParseError(
+            "Not a valid anchor: {}|{}|{}".format(
+                option, card_type, card_name
+                )
+            )
+    return anchor
+
+def getframe(option, card_type=None, card_name=None):
+    # Name of the option for the frame image
+    option = "frame_"+option if option else "frame"
+    return _find_and_load_image(option, card_type, card_name,
+                                get("frames directory", DEFAULT_SECTION))
+
+def getback(option, card_type=None, card_name=None):
+    # Name of the option for the back image
+    option = "back_"+option if option else "back"
+    return _find_and_load_image(option, card_type, card_name,
+                                get("frames directory", DEFAULT_SECTION))
+
+def getsymbol(option, card_type=None, card_name=None):
+    # Name of the option for the symbol image
+    option = "symbol_"+option if option else "symbol"
+    return _find_and_load_image(option, card_type, card_name,
+                                get("symbols directory", DEFAULT_SECTION))
+
+def getfont(option, card_type=None, card_name=None):
+    '''
+    Checks for a font object in the resources_cache dict. If it's not there,
+    checks the config file and attempts to load the font into the
+    resources_cache directory. 
+    '''
+    # Name of the option for the font object and filename
+    font_option = "font_"+option
+    fontfile_option = "fontfile_"+option
+    # Check resources_cache for font object
+    font = _get_resource(font_option, card_type, card_name)
+    if font is None:
+        # If not in resources_cache, check if the fontfile_ option exists in the
+        # config.
+        section = _find_option_in_config(fontfile_option,
+                                        card_type, card_name)
+        # If not, throw an error.
+        if section is None:
+            raise FontParseError("Font not found: {}|{}|{}".format(
+                option, card_type, card_name
+                ))
+        # Attempt to load a new font object from the config
+        font = _load_font(section, option,
+                          get("fonts directory", DEFAULT_SECTION))
+        # Add that new font object to the resources_cache object.
+        _add_to_resources(section, font_option, font)
+    return font
+
+def _find_option_in_config(option, card_type=None, card_name=None):
+    if config.has_section(card_name) and config.has_option(card_name, option):
```

Comment:
Good idea! I'd forgotten about this suggestion before, but this will go on the to-do list for when I get home.